common/scala/src/main/scala/whisk/core/containerpool/logging/DockerToActivationFileLogStore.scala

```diff
@@ -37,6 +37,7 @@ import spray.json._
 import spray.json.DefaultJsonProtocol._
 
 import scala.concurrent.Future
+import scala.concurrent.duration.DurationInt
```

Comment:
As a general rule of thumb: use `import scala.concurrent.duration._`, it makes it all easier